bgpd/bgp_updgrp_adv.c

```diff
@@ -533,6 +533,14 @@ void bgp_adj_out_unset_subgroup(struct bgp_node *rn,
 		if (adj->adv)
 			bgp_advertise_clean_subgroup(subgrp, adj);
 
+		/* If default originate is enabled and the route is default
+		 * route, do not send withdraw
+		 */
+		if (CHECK_FLAG(subgrp->sflags,
+			       SUBGRP_STATUS_DEFAULT_ORIGINATE) &&
```

Comment:
looks like a bad formatting?

Reply:
formatted in this way due to max chars allowed per line (80)

Reply:
that's fine, but it's not clang-format compatible, see @polychaeta comment